genai-perf/tests/test_json_exporter.py

```diff
@@ -358,3 +359,555 @@ def test_generate_json_custom_export(
         }
       }
     """
+
+    def test_valid_goodput_json_output(
```

Comment:
Since these test cases are just testing goodput, can you avoid making large data structs specific to goodput and just check the goodput-specific lines you expect are in the output? You don't need the other checks.